compiler/ccgtypes.nim

```diff
@@ -508,15 +508,15 @@ proc mangleRecFieldName(m: BModule; field: PSym): Rope =
 
 proc genRecordFieldsAux(m: BModule, n: PNode,
                         rectype: PType,
-                        check: var IntSet): Rope =
+                        check: var IntSet, union_prefix = ""): Rope =
```

Comment:
This should be ``unionPrefix`` and I have no idea why the style checker didn't catch this...